src/views/OrdersView.vue

```diff
@@ -326,34 +328,15 @@ export default Vue.extend({
     width: 7rem;
   }
 
-  .badge {
-    font-size: 0.9rem;
-    padding: $spacer $spacer * 2;
-  }
-
   .td-state {
-    overflow: visible !important;
-    width: 140px;
+    width: 9rem;
```

Comment:
add note on why 9 ( magic number)

Reply:
removed this style, its not really necessary anymore.